.github/workflows/validate.yml

```diff
@@ -0,0 +1,49 @@
+name: validate
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - main
+  pull_request:
+    branches:
+      - master
+      - main
+
+jobs:
+  linters:
+    permissions:
+      contents: read      # for actions/checkout to fetch code
+      pull-requests: read # for golangci/golangci-lint-action to fetch pull requests
+    timeout-minutes: 10
+
+    strategy:
+      matrix:
+        go-version: [1.19.x]
+        # FIXME(thaJeztah): enable once build-tags are fixed.
+        # os: [ubuntu-22.04, windows-2019]
+        os: [ubuntu-22.04]
```

Comment:
Windows can be enabled once https://github.com/vishvananda/netns/pull/61 is merged

Reply:
#61 has been merged, please add windows.